enacts/calc.py

```diff
@@ -718,7 +658,7 @@ def groupby_dekads(daily_data, time_dim="T"):
 
     See Also
     --------
-    xarray.groupby_bins
+    xarray.groupby_bins, xarray.DataArrayGroupBy, xarray.DatasetGroupBy
```

Comment:
`DataArrayGroupBy` and `DatasetGroupBy` aren't in the `xarray` package, they're in `xarray.core.groupby`.

Reply:
You resolved this comment without addressing it?

Reply:
boh...? I though I had...